_posts/2020-12-16-the-koji-making-process-temperature-mycelium-and-moisture.md

```diff
@@ -0,0 +1,200 @@
+
+![graph of temperature](/assets/images/koji_temp_curve/Graph 2.PNG){:height="450" .center}
+
+The entire process can roughly be broken down into three periods:
+
+1. For the first 18-22h, the rice is wrapped up and piled into a mound after inoculation *(toko-momi)*. This is done to keep the pile of rice warm and to retain as much moisture as possible. The koji spores germinate within 6-8 hours, then the pile of koji begins to warm up rapidly as the koji generates its own heat. In the middle of this period at the 10-12h mark, the koji is usually mixed once *(kiri-kaeshi)* to redistribute moisture between the inside and outside of the pile.
+2. A step called *mori* is performed after the initial 18-22h. The pile of rice is distributed into trays, boxes, or spread thinly on another table - depending on the koji-making method. The koji begins to generate a lot of heat at this point, so spreading it out helps dissipate heat and provide air. Two more mixing steps are usually performed at this stage *(naka-shigoto* and *shimai-shigoto)*.
+3. In the final phase of growth, the koji is well-established and enzyme production is the most active. At this stage, the temperature must be maintained according to the type of enzymes needed. Once complete, the act of removing the koji from the room and cooling it down is called *dekoji*
+
+A few caveats here:
+
+The graph above is for sake koji only, which finishes at a high temperature to produce a lot of amylase and suppress proteases. I've compiled all the temperature curves I have [here](https://imgur.com/a/q94PGBw). Sometimes there will be only two distinct temperature phases if the temperature requirements of periods 2. and 3. are identical.
+
+The exact shape and slope of the curve also depends on the koji-making method (more on methods [here](https://controlledmold.com/industrial-koji-fermentation-equipment/)). Machine-made koji will have very accurate temperature control, while traditional and manual methods will show a lot more variability and rely on frequent mixing to bring the temperature down.
+
+And lastly, temperature is just one part of the picture. As the koji progresses, two things need to happen:
+
+1. The koji mycelium needs to grow on the **surface** and **interior** of each grain
+
+2. The koji needs to have a proper amount of **moisture**
+
+### Mycelia and "Haze"
+
+Japanese brewers have a term called *"haze"* (pronounced like *saké*) that describes the state of mycelial growth growing on a grain of koji. The term is further distinguished into two concepts:
+
+1. *Haze-mawari* - how to koji mycelium spreads on a grain's **surface**
+2. *Haze-komi* - how the koji mycelium penetrates into the grain's **interior**
+
+Koji with a high enzyme concentration must have good growth on the surface of the grain, the centre of the grain, and have the proper moisture content. Brewers have different terms to describe the final product koji's visible surface and interior growth:
+
+{:.table-clean}
```

Comment:
using this class, the tables appear more "table-like":

<img width="891" alt="Screen Shot 2020-12-16 at 10 38 30 AM" src="https://user-images.githubusercontent.com/884032/102370239-d8c8b100-3f8a-11eb-8fe1-c203472ad717.png">

Reply:
Noted - thanks. Looks good.